README.md

```diff
@@ -65,7 +65,7 @@ the `data` directory; you can access it at `/data/<filename>.json`.
 
 ### Available dependencies
 
-The repo includes jQuery, Backbone, Underscore, and RequireJS. If there's other
+The repo includes jQuery, Backbone, and Underscore. If there's other
 stuff you'd find useful, you can put it in the `lib` directory.
```

Comment:
I thought about adding that, but decided against adding this to this particular section. It seems to me that the current tools provided are more than enough to solve the current tests.

I could see instructions on adding new tools belonging in the "I don't like this tech stack" / fork this project section, though.

What do you think? :open_mouth: